packages/Python/lldbsuite/test/functionalities/data-formatter/data-formatter-stl/libstdcpp/unique_ptr/TestDataFormatterStdUniquePtr.py

```diff
@@ -16,6 +16,7 @@ class StdUniquePtrDataFormatterTestCase(TestBase):
     mydir = TestBase.compute_mydir(__file__)
 
     @skipIfDarwin  # doesn't compile on Darwin
+    @skipIfLinux
```

Comment:
I was looking at the same failure and I wonder whether we can make this more targeted? Overall, the test seems to pass on other bots (e.g. upstream-with-swift 16.04, unless I'm mistaken).

Reply:
Oh, this is 4.2, probably not worth it.